tools/onenote-migration/test_migrate.py

```
@@ -0,0 +1,300 @@
+
+
+class TestHtmlToMarkdown(unittest.TestCase):
+    def test_heading_conversion(self):
+        self.assertIn("# Hello", M.html_to_markdown("<h1>Hello</h1>"))
+    def test_paragraph(self):
+        self.assertIn("Some text", M.html_to_markdown("<p>Some text</p>"))
+    def test_bold(self):
+        self.assertIn("Bold", M.html_to_markdown("<b>Bold</b>"))
+    def test_list(self):
+        md = M.html_to_markdown("<ul><li>Item A</li><li>Item B</li></ul>")
+        self.assertIn("Item A", md); self.assertIn("Item B", md)
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)